.github/workflows/validate.yml

```diff
@@ -138,3 +138,8 @@ jobs:
       run: make validate
     - name: make release
       run: make release
+    - name: upload release
```

Comment:
I would call this as "artifact" rather than "release"

Reply:
fixed